packages/fluentui/e2e/tests/treeVirtualizedKeyboardNavigation-example.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { VirtualizedTreePrototype } from '@fluentui/react-northstar-prototypes';
+import { Loader } from '@fluentui/react-northstar';
+
+const treeVirtualizedKeyboardNavigation = () => (
+  <React.Suspense fallback={<Loader />}>
```

Comment:
```suggestion
  <React.Suspense fallback={<div />}>
```

nit, this can be just `div` as we don't care about UI there